src/app/TideSearchApplication.cpp

```diff
@@ -400,7 +400,7 @@ int TideSearchApplication::main(const vector<string>& input_files, const string
   return 0;
 }
 
-vector<int> TideSearchApplication::getNegativeIsotopeErrors() const {
+vector<int> TideSearchApplication::getNegativeIsotopeErrors() {
```

Comment:
why did you remove const?

Reply:
because I make the function static which can be called elsewhere